ecommerce/baskets/filters.py

```diff
@@ -0,0 +1,32 @@
+from django.utils.translation import gettext_lazy as _
+from django_filters import rest_framework as filters
+
+from orders import enums
+from baskets.models import Basket, BasketItem
+from customers.filters import CustomerFilter
+from products.filters import ProductFilter
+
+
+class BasketFilter(filters.FilterSet):
+    """
+     Filter for BasketViewSet
+    """
+    customer = CustomerFilter
```

Comment:
bu kullanimi nereden ogrendiniz? ve query parametreleri icerisinde nasil kullaniyoruz?

Reply:
Bu kullanımı (sahinof-hw-3)[https://github.com/upy/bootcamp/pull/70]'te görüp nested filtre oluşturmakta kullanabilir miyim diye düşünerek eklemiştim. Ne yazık ki çalıştıramadım. Silmeyi atlamışım. Yerine doğru nested filtreler eklendi. Düzeltme için teşekkür ederim.